src/container/srv_target.c

```diff
@@ -1411,6 +1426,10 @@ ds_cont_local_open(uuid_t pool_uuid, uuid_t cont_hdl_uuid, uuid_t cont_uuid,
 			poh = hdl->sch_cont->sc_pool->spc_hdl;
 			rc = 0;
 		}
+
+		d_list_add_tail(&hdl->sch_link, &cont->sc_open_hdls);
```

Comment:
Suggestion: add and remove the hdl from sc_open_hdls list in cont_hdl_add() and cont_hdl_delete().